src/main/resources/log4j2.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
```

Comment:
A couple of questions:
1) Will this (and if not, could we) suppress anything less severe than `WARN` coming through log4j?
2) Could you please add a comment to the file that this exists largely to configure/suppress logging for the intel library (and maybe some other external libraries?) please?

Reply:
I was still seeing INFO logs coming through, but I will try to adjust it so that is not the case.

Reply:
Ah actually this is doing that.. just sending them back out at INFO level.